File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q11.js
import { question } from "readline-sync";
function main(){
    //Input
    const num = (question("Enter a number with 3 digits: "))

    //Processing
    let hundreds = Math.floor(num / 100)
    let dozens = Math.floor((num % 100)/10)
    let units = (num % 100) % 10

    //Output
    console.log(`The entered number ${num} in reverse order is: ${units}${dozens}${hundreds}`)
}
    

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q11.py
def main():
    #Input
    num = int(input("Enter a number: "))

    #Processing
    hundreds = round(num / 100)
    dozens = round((num % 100)/10)
    units = (num % 100)%10

    #Output
    print(f"The reverse of the {num} is: {units}{dozens}{hundreds}")

main()